.gitignore

```diff
@@ -45,6 +45,7 @@ Session.vim
 /_tmp/
 *.un~
 .vagrant
+vendor
```

Comment:
```suggestion
```

Reply:
I added this intentionally to ensure that the contributors don't end up committing the vendor/ (it seemed like only a build-time req. anyway).

Should I still remove it from gitignore?

Reply:
This need not block the PR, but we can likely skip this update in this PR, and if required send an independent one. Reason being, I actually don't recall why vendor was never added to .gitignore (although we infact don't commit vendor) -- this _may_ just turn into another long discussion which may block this PR even longer.

Reply:
Gotcha makes sense!